src/net_processing.cpp

```diff
@@ -488,6 +488,14 @@ struct Peer {
     /** Whether this peer should be disconnected and marked as discouraged (unless it has the noban permission). */
     bool m_should_discourage GUARDED_BY(m_misbehavior_mutex){false};
 
+    /** Set of txids to reconsider once their parent transactions have been accepted **/
+    std::set<uint256> m_orphan_work_set GUARDED_BY(g_cs_orphans);
+
+    /** Protects vRecvGetData **/
+    Mutex cs_vRecvGetData;
+    /** Work queue of items requested by this peer **/
+    std::deque<CInv> vRecvGetData GUARDED_BY(cs_vRecvGetData);
```

Comment:
Since you're touching every line that `vRecvGetData` is on already, you may as well add a scripted-diff commit to this branch that updates the name to modern style guidelines, e.g. something like `m_get_data_requests` or similar. Same for the mutex name - it could be renamed to `m_get_data_requests_mutex` or similar.

Reply:
done